internal/provider/resource_vdb_test.go

```diff
@@ -88,6 +121,43 @@ func testAccCheckDctVdbResourceExists(n string) resource.TestCheckFunc {
 	}
 }
 
+func testAccCheckDctVdbBookmarkResourceExists(n string) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		rs, ok := s.RootModule().Resources[n]
+
+		if !ok {
+			return fmt.Errorf("Not found: %s", n)
+		}
+
+		vdbId := rs.Primary.ID
+		if vdbId == "" {
+			return fmt.Errorf("No VdbID set")
+		}
+
+		client := testAccProvider.Meta().(*apiClient).client
+
+		res, _, err := client.VDBsApi.GetVdbById(context.Background(), vdbId).Execute()
```

Comment:
I would avoid using variable names like `res` and `res2` and prefer context specific names.

Reply:
done